provider/junit/README.md

```diff
@@ -393,6 +393,8 @@ public class PactJUnitTest {
 }
 ```
 
+**NOTE!** You will not be able to publish the verification results when you apply a filter.
```

Comment:
Is it worth specifying that it only happens if you leave one or more interaction outside of your filters?

Potentially you can have multiple filters that cover all the interactions and in that case the verification results will be published, right?

Reply:
Hmmm.. I realized I tested this in the 3.5.x branch (because I was investigating another bug). It might no longer be the case in 4, I will check.

Reply:
I extended the description accordingly.

Reply:
Brilliant, thanks for looking into that, are you going to open an issue about the publication bug with junit5 ?

Reply:
I will take a look to see why that's happening (if you are not already planning to do so yourself)

Reply:
Yes, looking into fixing it right now, will either create issue + PR or just the issue if I cannot make any progress.

Reply:
Okay :+1:  :+1:  :+1: